### PR TITLE
Add package for bisect-summary

### PR DESCRIPTION
**Pull request by lindig**
First packaging for Opam. Don't be confused by the 0.2 in the file name of the ZIP file. It refers to a slightly older version than the current one on GitHub.

Signed-off-by: Christian Lindig <lindig@gmail.com>

### CODE REVIEW

File: packages/bisect-summary/bisect-summary.0.3/url [hunk 1]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/lindig/bisect-summary/archive/0.2.zip"
```
Comment (dsheets) [inline packages/bisect-summary/bisect-summary.0.3/url]: This archive URL says 0.2 but the package is 0.3. Is this correct?

File: packages/bisect-summary/bisect-summary.0.3/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
+opam-version   : "1.2"
+name           : "bisect-summary"
+version        : "0.3"
+maintainer     : "Christian Lindig <lindig@citrix.com>"
+authors        : [  "Christian Lindig <lindig@citrix.com>"]
+license        : "MIT"
+homepage       : "https://github.com/lindig/bisect-summary"
+bug-reports    : "https://github.com/lindig/bisect-summary"
```
Comment (dsheets) [inline packages/bisect-summary/bisect-summary.0.3/opam]: Usually, the `bug-reports` field is the GitHub `/issues` page.

File: packages/bisect-summary/bisect-summary.0.3/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
+opam-version   : "1.2"
+name           : "bisect-summary"
+version        : "0.3"
```
Comment (dsheets) [inline packages/bisect-summary/bisect-summary.0.3/opam]: Please don't include the `name` and `version` fields in the opam-repository metadata. `opam-publish` will remove these fields for you automatically to make a pinnable `opam` and a publishable `opam` easier to maintain (we know this is annoying).

File: packages/bisect-summary/bisect-summary.0.3/url (hunk 1 above)
Comment (lindig) [inline packages/bisect-summary/bisect-summary.0.3/url]: Yes. I said so in the top comment.

Comment (dsheets) [inline packages/bisect-summary/bisect-summary.0.3/url]: I am confused (even though you told me not to be). You have both `0.2` and `0.3` tagged in your repo but you want `0.3` in opam to actually be `0.2` on your repo? Isn't that going to confuse users?

Comment (lindig) [inline packages/bisect-summary/bisect-summary.0.3/url]: The URL now points to the 0.3 version. This was rooted in the (at least for me annoying) inability to have all opam-related records in the same repo when hosting on GitHub: you can't know the checksum of the release you are about to tag.

**Comment by dsheets** [comment]
Thanks!

**Pull request merged**